### PR TITLE
[Merged by Bors] - Set up tools and actions

### PR DESCRIPTION
**Pull request by teozkr**

### CODE REVIEW

**Comment by teozkr** [comment]
bors try

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
```
Comment (maltesander) [inline .github/workflows/gradle.yml:23]: Im not sure about java 11 and druid. I think we should use 8 here. @fhennig what do you think?

Comment (fhennig) [reply .github/workflows/gradle.yml:23]: Ah, just saw this now. Yes the stuff needs to be compiled with source compatiblity 1.8, but it should work with the JDK 11 anyways

Comment (maltesander) [reply .github/workflows/gradle.yml:23]: Then its good to go! Maybe activate branch protection @teozkr?

Comment (teozkr) [reply .github/workflows/gradle.yml:23]: Can't activate the branch protection before at least one PR has been merged by Bors.

**Comment by teozkr** [comment]
bors retry

**Comment by teozkr** [comment]
bors retry

**Comment by teozkr** [comment]
bors retry

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (teozkr) [reply .github/workflows/gradle.yml:23]: Branch protection is now enabled.

**Pull request closed without merging**